packages/pluggable-widgets-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/pluggable-widgets-tools",
-  "version": "9.23.2",
+  "version": "9.24.0",
```

Comment:
Please no, this will just block my PR

Reply:
And there is no changes in piw-tools to receive a bump

Reply:
This was done so I could trigger the Script tests and see if the solution works.

Should we keep it until everyone reviews the PR and then before merging I remove it?